src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
That is true, but on the other hand

```scala
scala> val s = collection.mutable.Seq(1)
s: scala.collection.mutable.Seq[Int] = ArrayBuffer(1)

scala> s += 2
<console>:12: error: value += is not a member of scala.collection.mutable.Seq[Int]
       s += 2
         ^
```

so people who are going by the API and not by pattern-matching can't rely on it even now.  And that `Seq` might very well be a `WrappedArray` that doesn't have the operation now.

So I think there's a decent argument for leaving it alone, too.  But having `+=` as a core functionality (and one that was usually lurking there one pattern match away, even if you couldn't count on it) is appealing too, so `FixedLengthSeq` (or maybe `SizedSeq` so it's not too wordy?) is also reasonable.